src/components/PopupConfirmDelete.js

```diff
@@ -0,0 +1,21 @@
+import Popup from "./Popup.js";
+
+export default class PopupConfirmDelete extends Popup {
```

Comment:
Great job implementing confirmation popup and the functionality related to it! In general, it can support any confirmation.